src/web/mjs/connectors/DynastyScans.mjs

```diff
@@ -26,14 +26,17 @@ export default class DynastyScans extends Connector {
      */
     _getMangaList( callback ) {
         let promises = ['/series', '/anthologies', '/issues', '/doujins'].map( page => {
-            return this.fetchDOM( this.url + page, '.tag-list dd a' )
-                .then( data => {
-                    let mangaList = data.map( element => {
-                        return {
-                            id: this.getRelativeLink( element ),
-                            title: element.text.trim()
-                        };
-                    } );
+            return this.fetchJSON( this.url + `${page}.json` )
```

Comment:
Tip: In the current HakuNeko versions you can also use async/await pattern, which makes the promises more readable.
Example: https://github.com/manga-download/hakuneko/wiki/Developer-Website-Tutorial#47-from-callbacks-over-promises-to-asyncawait

Reply:
Noted, thanks.
I won't have time to make the modifications tonight, I'll do it ASAP and push again !